code/datums/inventory_slots/inventory_gripper.dm

```diff
@@ -19,18 +24,31 @@
 		var/obj/item/organ/external/E = GET_EXTERNAL_ORGAN(owner, slot_id)
 		return "[pronouns.He] [pronouns.is] holding [_holding.get_examine_line()] in [pronouns.his] [E?.name || lowertext(slot_name)]."
 
+/datum/inventory_slot/gripper/can_equip_to_slot(var/mob/user, var/obj/item/prop, var/disable_warning)
+	return ..() && user.check_dexterity(DEXTERITY_GRIP)
+
 // Hand subtypes below
-// Mouths are used by diona nymphs and Ascent babies to eat stuff, not just hold stuff in the mouth.
 /datum/inventory_slot/gripper/mouth
 	slot_name = "Mouth"
 	slot_id = BP_MOUTH
-	requires_organ_tag = null
+	requires_organ_tag = BP_HEAD
 	can_use_held_item = FALSE
+	overlay_slot = BP_MOUTH
+	ui_label = "M"
+	hand_sort_priority = 3
+
+/datum/inventory_slot/gripper/mouth/get_examined_string(mob/owner, mob/user, distance, hideflags, decl/pronouns/pronouns)
+	if(_holding)
+		return "[pronouns.He] [pronouns.is] holding [_holding.get_examine_line()] in [pronouns.his] mouth."
```

Comment:
Is this override necessary? `E?.name || lowertext(slot_name)` should work fine even with no 'mouth' organ, and `lowertext(slot_name)` would certainly work without hardcoding `mouth`.

Reply:
For grippers:
`return "[pronouns.He] [pronouns.is] holding [_holding.get_examine_line()] in [pronouns.his] [E?.name || lowertext(slot_name)]."`